tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -696,7 +714,9 @@
           && selectedDashboard == null);
       },
       _computeShowNoSuchDashboardMessage(dashboards, selectedDashboard) {
-        return !!selectedDashboard && !dashboards.includes(selectedDashboard);
+        return !!selectedDashboard &&
```

Comment:
Can we just write `!!selectedDashboard && !dashboards.some(d => d.plugin === selectedDashboard)`? The lodash boilerplate is really unnecessary here.

Reply:
Yes - that boilerplate looks convoluted. Done.